WordPress/Classes/ViewRelated/Post/PostNoticeNavigationCoordinator.swift

```diff
@@ -23,23 +23,13 @@ class PostNoticeNavigationCoordinator {
                 return
         }
 
-        if FeatureFlag.postPreview.enabled {
-            let controller = PostPreviewViewController(post: page)
-            controller.navigationItem.title = NSLocalizedString("View", comment: "Verb. The screen title shown when viewing a post inside the app.")
-            controller.hidesBottomBarWhenPushed = true
+        let controller = PreviewWebKitViewController(post: page)
```

Comment:
I think this was a bug in the original code, since we were showing PostPreviewViewController when FeatureFlag.postPreview.enabled. So I just left the else condition instead.